packages/build/src/plugins_core/functions/index.js

```diff
@@ -63,6 +68,22 @@ const getZisiParameters = ({ buildDir, featureFlags, functionsConfig, functionsD
   return { basePath: buildDir, config, manifest, featureFlags: zisiFeatureFlags }
 }
 
+// TODO: is this the best place for this logic? should this live elsewhere, e.g. in its own plugin?
+const writeToScheduleFile = async (zisiResult) => {
```

Comment:
@eduardoboucas WDYT, where should this logic live?

Reply:
Yes, `plugins_core/functions/utils.js` makes sense :+1: